### PR TITLE
Update support notifications to use common boostrap logic.

### PR DESCRIPTION
**Pull request by brandonforster**
Update main.go and init.go to leverage the common bootstrapping functionality available to all services and remove redundant code.

Signed-off-by: Brandon Forster <brandonforster@gmail.com>

### CODE REVIEW

File: cmd/support-notifications/main.go [hunk 1]
```diff
@@ -16,29 +16,29 @@
...
 package main
 
 import (
 	"flag"
-	"fmt"
-	"os"
-	"os/signal"
-	"strconv"
-	"time"
+	"github.com/edgexfoundry/edgex-go/internal/support/notifications"
```
Comment (michaelestrin) [inline cmd/support-notifications/main.go]: Clean up imports.

File: internal/support/notifications/init.go [hunk 2]
```diff
@@ -18,116 +18,43 @@
...
-		return false
-	}
-	if useRegistry {
-		registryErrors = make(chan error)
-		registerUpdates = make(chan interface{})
-		go listenForConfigChanges()
-	}
-
-	go telemetry.StartCpuUsageAverage()
+var chEvents chan interface{} // A channel for "domain events" sourced from event operations
```
Comment (michaelestrin) [inline internal/support/notifications/init.go]: Initialized but otherwise unused. Remove.

File: internal/support/notifications/init.go [hunk 3]
```diff
@@ -150,110 +77,51 @@ func newDBClient(dbType string) (interfaces.DBClient, error) {
...
-func setLoggingTarget() string {
-	if Configuration.Logging.EnableRemote {
-		return Configuration.Clients["Logging"].Url() + clients.ApiLoggingRoute
-	}
-	return Configuration.Logging.File
+		LoggingClient.Info("Database disconnected")
+	}()
+
+	// initialize event handlers
+	chEvents = make(chan interface{}, 100)
```
Comment (michaelestrin) [inline internal/support/notifications/init.go]: support-notifications has no event handlers.  Remove this code.

File: internal/support/notifications/init.go [hunk 4]
```diff
@@ -150,110 +77,51 @@ func newDBClient(dbType string) (interfaces.DBClient, error) {
...
-		if configuration.Service.Port == 0 {
-			return nil, errors.New("error reading configuration from Registry")
+	// initialize database.
+	for startupTimer.HasNotElapsed() {
+		var err error
+		dbClient, err = s.newDBClient(Configuration.Databases["Primary"].Type)
+		if err == nil {
+			break
 		}
+		LoggingClient.Warn(fmt.Sprintf("couldn't create database client: %v", err.Error()))
```
Comment (michaelestrin) [inline internal/support/notifications/init.go]: You need a dbClient = nil statement before the logging statement. See #1827

File: internal/support/notifications/config.go [hunk 5]
```diff
@@ -14,7 +14,10 @@
  *******************************************************************************/
 package notifications
```
Comment (michaelestrin) [inline internal/support/notifications/config.go]: While you're in here, please add an empty line between comment and package statement.

**Comment by michaelestrin** [comment]
recheck

**Comment by brandonforster** [comment]
recheck

**Comment by brandonforster** [comment]
recheck

**Pull request merged**